gui/wxpython/nviz/wxnviz.py

```diff
@@ -289,7 +289,8 @@ def print_progress(value):
 
 
 class Nviz:
-    def __init__(self, glog, gprogress):
+
+    def __init__(self, glog, gprogress) -> None:
```

Comment:
glog and gprogress are not annotated... Is that on purpose?

Reply:
I wanted to have the width and height constrained, as it was needed somewhere else, but mypy does not check the bodies of untyped functions, by default, (some other tools do). I didn't analyze or didn't have enough info yet for the other two arguments to investigate all possible calls.

Typing in python is gradual, and it's fine, otherwise everything would need to be changed at once, as one info added helps another place, just like a sudoku.